### PR TITLE
Updating docs to reflect removal of timer APIs and addition of certain new APIs

### PR DESCRIPTION
**Pull request by ArGup**
## Description

- Updating the docs to reflect the removal of timer Web APIs.
- Adding a new API `queueAsyncEdit` in Editor class.
- Other API updates

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by hollyschinsky — CHANGES_REQUESTED** [review]
Hi @ArGup - for any new classes, interfaces, or enums, please add their references into the [gatsby-config.js](https://github.com/AdobeDocs/express-add-ons-docs/blob/main/gatsby-config.js) (in the same way the others are) so they show up in the left side navigation.

File: src/pages/references/document-sandbox/document-apis/classes/LineNode.md [hunk 1]
```diff
@@ -14,38 +14,37 @@ A LineNode represents a simple line object in the scenegraph – a single straig
...
 
-- [DEFAULT\_END\_X](LineNode.md#DEFAULT_END_X)
-- [DEFAULT\_END\_Y](LineNode.md#DEFAULT_END_Y)
-- [DEFAULT\_START\_X](LineNode.md#DEFAULT_START_X)
-- [DEFAULT\_START\_Y](LineNode.md#DEFAULT_START_Y)
-- [DEFAULT\_STROKE\_WIDTH](LineNode.md#DEFAULT_STROKE_WIDTH)
+- [DEFAULT\_END\_X](LineNode.md#default_END_X)
+- [DEFAULT\_END\_Y](LineNode.md#default_END_Y)
+- [DEFAULT\_START\_X](LineNode.md#default_START_X)
+- [DEFAULT\_START\_Y](LineNode.md#default_START_Y)
```
Comment (peterflynn) [inline src/pages/references/document-sandbox/document-apis/classes/LineNode.md:20]: Is this capitalization correct? Should the entire anchor name be lowercase?

File: src/pages/references/document-sandbox/document-apis/classes/Node.md [hunk 2]
```diff
@@ -188,111 +156,150 @@ The node's parent. Undefined if the node is an orphan, or if the node is the art
...
 
-• `get` **relativeTransform**(): [`mat2d`](https://glmatrix.net/docs/module-mat2d.html)
+• `get` **transformMatrix**(): `https://glmatrix.net/docs/module-mat2d.html`
 
 The node's transform matrix relative to its parent.
 
 #### Returns
 
-[`mat2d`](https://glmatrix.net/docs/module-mat2d.html)
+`https://glmatrix.net/docs/module-mat2d.html`
```
Comment (peterflynn) [inline src/pages/references/document-sandbox/document-apis/classes/Node.md:193]: Looks like somehow we lost the clickable links here. Compare to the markup for the old getter:


```md
### relativeTransform

• `get` **relativeTransform**(): [`mat2d`](https://glmatrix.net/docs/module-mat2d.html)

The node's transform matrix relative to its parent.

#### Returns

[`mat2d`](https://glmatrix.net/docs/module-mat2d.html)
```

Comment (hollyschinsky) [reply src/pages/references/document-sandbox/document-apis/classes/Node.md]: @peterflynn @ArGup fixed all the links and committed back.

File: src/pages/references/changelog.md [hunk 3]
```diff
@@ -226,7 +226,7 @@ Added new code sample to demonstrate how to use SWC-React and set theme properti
...
   - It's occasionally possible to run into a race condition where the communications bridge between the two contexts (panel vs document sandbox) is not set up in time. If you interact with your panel UI immediately after it's reloaded, the click may appear do nothing instead of invoking your script code. We're working on a fix for this.
 - Common pitfalls
-  - If you split your work on a document over multiple frames using `setTimeout`, be sure to protect against reentrancy, otherwise you may end up corrupting the user's undo stack. You should disable elements on the panel UI that could allow the user to execute your code before it is complete and then re-enable those elements when the code is done. The issue will be fixed in a future release.
+  - If you split your work on a document over multiple frames, be sure to protect against reentrancy, otherwise you may end up corrupting the user's undo stack. You should disable elements on the panel UI that could allow the user to execute your code before it is complete and then re-enable those elements when the code is done. The issue will be fixed in a future release.
```
[the remark this answers is not in the document]
Comment (hollyschinsky) [reply src/pages/references/changelog.md:262]: Agree, will work on in a follow-on.

**Review by hollyschinsky — APPROVED** [review]
Fixed the mat2d links and the linting errors around the fenced code blocks and others. Be sure to use a blank line before and after any code blocks as well as indicate the language going forward to avoid those errors. For instance, this block will error:

Center a rectangle within its parent artboard:
```
rectangle.setPositionInParent(
    { x: artboard.width / 2, y: artboard.height / 2 },
    { x: rectangle.width / 2, y: rectangle.height / 2 }
);
```

But this won't:

Center a rectangle within its parent artboard:

```js
rectangle.setPositionInParent(
    { x: artboard.width / 2, y: artboard.height / 2 },
    { x: rectangle.width / 2, y: rectangle.height / 2 }
);
```

There's one minor one left to fix in changelog but merging because it doesn't cause any issues in the display.

**Pull request merged**